core/src/main/scala/chisel3/Data.scala

```diff
@@ -488,6 +547,14 @@ abstract class Data extends HasId with NamedComponent with SourceInfoDoc {
     }
     requireIsHardware(this)
     topBindingOpt match {
+      // DataView
+      case Some(ViewBinding(target)) => reify(target).ref
+      case Some(AggregateViewBinding(viewMap, _)) =>
+        viewMap.get(this) match {
+          case None => materializeWire() // FIXME FIRRTL doesn't have Aggregate Init expressions
```

Comment:
Is this something to be fixed in a future PR?

Reply:
This `FIXME` is copy-pasted from the same thing in surrounding lines for BundleLits, VecLits, and DontCare.

We chatted in person but I think the ideal approach is to have aggregate init expressions in FIRRTL itself, or at least add them to Chisel IR and turn them into Wires during ChiselIR => Chirrtl (ie. in the Converter).

This is unrelated to this PR though, but this work would benefit from that.